_config.yml

```diff
@@ -35,9 +35,9 @@ tierthreeplan: Tiny Cloud Professional Plan
 enterpriseplan: Tiny Custom Plans
 ## common tiny.cloud URLs
 supporturl: https://support.tiny.cloud
-plugindirectory: https://www.tiny.cloud/apps/
+plugindirectory: https://www.tiny.cloud/tinymce/features/
 gettiny: https://www.tiny.cloud/get-tiny
-accountpageurl: https://apps.tiny.cloud/my-account
+accountpageurl: https://www.tiny.cloud/my-account
```

Comment:
we should add trailing slashes/ for all the links, it should save on 301 redirects

Reply:
in the current system, baseurl and url are set by the site, such as:
```html
<body role="document" data-shared-baseurl="" data-baseurl="/docs">
```
so if we do this, some will have the trailing slash, others wont; which could get confusing. Hopefully we will eventually fix this.